src/main/java/io/snyk/jenkins/CustomBuildToolPathCallable.java

```diff
@@ -20,6 +21,7 @@
 
 class CustomBuildToolPathCallable implements FilePath.FileCallable<String> {
 
+  @Serial
```

Comment:
Java 17 is now the supported baseline for Jenkins. `@Serial` is a Java 17 feature (introduced in Java 14). More details: https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/io/Serial.html